src/main/java/org/folio/services/budget/BudgetExpenseClassTotalsService.java

```diff
@@ -78,8 +77,7 @@ private BudgetExpenseClassTotalsCollection buildBudgetExpenseClassesTotals(List<
   }
 
   private double getBudgetTotalExpended(Budget budget) {
```

Comment:
We don't need this method anymore, this can be simplified.

Reply:
regarding name, it is okay to be totalExpended?

Reply:
method removed, thanks for noticing